lib/tasks/app.rake

```diff
@@ -28,12 +30,16 @@ namespace :app do
   def restore_cropstore_dump
     db_config = Rails.application.config_for(:database)
     env = { "PGUSER" => db_config["username"], "PGPASSWORD" => db_config["password"] }
-    cmd = "pg_restore -O -d #{db_config["database"]} db/cropstore_web_20150403_1530.dump"
+    cmd = "pg_restore -O -d #{db_config["database"]} db/cropstore_dev_20150403_1530.dump"
```

Comment:
I renamed the file itself, not the reference in `app.rake`. It won't merge properly.